### ISSUE TITLE
Add functional.dropout

### ISSUE THREAD

**Issue by blue-season** [blue-season/pywarm#1]
Create one functional.dropout to unify nn.Dropout, nn.Dropout2d, nn.Dropout3d.

Note that nn.Dropout is not restricted to 1d cases. Therefore, a flag will be used to determine if the dropout is applied randomly to all dimensions or entire channels.

**Comment by blue-season** [comment]
Fixed in #2 .
